provisioning/transport/mqtt/src/ProvisioningTransportHandlerMqtt.cs

```diff
@@ -77,13 +77,25 @@ public class ProvisioningTransportHandlerMqtt : ProvisioningTransportHandler
 
             cancellationToken.ThrowIfCancellationRequested();
 
-            SecurityProviderX509 security;
-
+            RegistrationOperationStatus operation = null;
             try
             {
                 if (message.Security is SecurityProviderX509)
                 {
-                    security = (SecurityProviderX509)message.Security;
+                    SecurityProviderX509 x509Security = (SecurityProviderX509)message.Security;
```

Comment:
This seems to be incorrect.

Reply:
use:

if (message.Security is SecurityProviderX509 securityProvider) 

---
In reply to: [217852616](https://github.com/Azure/azure-iot-sdk-csharp/pull/619#discussion_r217852616) [](ancestors = 217852616)

Reply:
Yes, doesn't look like it's needed

---
In reply to: [217852616](https://github.com/Azure/azure-iot-sdk-csharp/pull/619#discussion_r217852616) [](ancestors = 217852616)